scripts/seed-data/capture-analytics-state.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+echo "📊 Capturing Matomo Analytics Golden State"
+echo "==========================================="
+
+# Check if containers are running
+if ! docker compose ps mysql | grep -q "Up"; then
+  echo "❌ mysql container is not running"
+  echo "Start it with: npm start"
+  exit 1
+fi
+
+if ! docker compose ps analytics-zoo | grep -q "Up"; then
+  echo "❌ analytics-zoo container is not running"
+  echo "Start it with: docker compose --profile on-demand up -d analytics-zoo"
+  exit 1
+fi
+
+# Create directories
+mkdir -p sites/apps/analytics.zoo/data-golden
+
+echo ""
+echo "1️⃣ Capturing MySQL database state..."
+docker exec the_zoo-mysql-1 mysqldump \
+  -u analytics_user \
+  -panalytics_pw \
+  analytics_db \
+  > core/mysql/sql/analytics_seed.sql
+
+echo "✅ Database dump saved to core/mysql/sql/analytics_seed.sql"
+echo "   Size: $(du -h core/mysql/sql/analytics_seed.sql | cut -f1)"
+
+echo ""
+echo "2️⃣ Capturing Matomo application data..."
```

Comment:
Does this config folder contain seed-specific state? We have to do that on gitea bc it stores the actual repos on disc, but for many sites it's not necessary (if the web server is more or less stateless)